include/GafferImage/ImageReader.h

```diff
@@ -110,10 +110,18 @@ class ImageReader : public ImageNode
 		Gaffer::StringPlug *colorSpacePlug();
 		const Gaffer::StringPlug *colorSpacePlug() const;
 
+		Gaffer::StringPlug *colorSpaceMetadataNamePlug();
```

Comment:
I have no recollection of either being keen on CompoundObject nor why I might have been...

Off the top of my head, could it be because CompoundDataPlug is trying to expose its members as child plugs and we don't have an AtomicCompoundDataPlug ?